docling_core/types/doc/document.py

```diff
@@ -591,7 +591,13 @@ def export_to_markdown(self) -> str:
         for row in self.data.grid:
             tmp = []
             for col in row:
-                tmp.append(col.text)
+
+                # make sure that md tables are not broken
+                # due to newline chars in the text
+                text = col.text
+                text = text.replace("\n", "")
```

Comment:
It would be better to have a separator, at least `" "`, because some other tables might be `TEXT\nSUBTEXT`

Technically, the better replacement (only in Markdown) could be `<br />`.

Reply:
good catch, that was my intention in the first place ... must have slipped